src/Microsoft.Azure.ServiceBus/Amqp/AmqpLinkCreator.cs

```diff
@@ -78,6 +78,7 @@ protected AmqpLinkCreator(string entityPath, ServiceBusConnection serviceBusConn
                     amqpConnection,
                     exception);
 
+                session.Abort();
```

Comment:
Call SafeClose instead. Abort does not notify the remote peer of state change.

Reply:
@xinchen10  So should it be updated for exceptions while creating session? In line 61 above